tox.ini

```diff
@@ -22,6 +20,8 @@ extras =
     test
 
 commands =
+    python -V
+    pip list
```

Comment:
To my mind these changes are only visual noise. (I could live with the Python version, but `pip list` seems to be to heavy.)

Reply:
it may be visual noise, but helps to debug or clearify which Packages are actually installed in the test run.

Reply:
Okay, let's keep this change.